### ISSUE TITLE
Cf-exporter not connecting to PCF

### ISSUE THREAD

**Issue by aparnareji** [bosh-prometheus/cf_exporter#22]
Hi Team, 

I have deployed cf-exporter in my environment and need to get metrics from Pivotal Cloud Foundry. 
I have provided the following Environmental Variables:
```
CF_EXPORTER_CF_API_URL : https://api.sys.adp.allianz/
CF_EXPORTER_CF_CLIENT_ID : <client-id >
CF_EXPORTER_CF_CLIENT_SECRET : <client-secret >
CF_EXPORTER_METRICS_ENVIRONMENT : pcf-adp
```
But with this environmental variables , I have found that , it is not connecting to `pivotal cloud foundry` that we are not getting the metrics for that . Please find the attached `error` we got in `logs`:
```

  | time="2018-02-19T07:01:22-05:00" level=info msg="Starting cf_exporter (version=, branch=, revision=)" source="cf_exporter.go:127"
-- | --
  | time="2018-02-19T07:01:22-05:00" level=info msg="Build context (go=go1.8, user=, date=)" source="cf_exporter.go:128"
  | time="2018-02-19T07:01:23-05:00" level=info msg="Listening on :9193" source="cf_exporter.go:221"
  | time="2018-02-19T07:02:17-05:00" level=error msg="Error while listing service instances: Error requesting service instances: cfclient: error (10000): CF-NotFound" source="service_instances_collector.go:142"
  | time="2018-02-19T07:02:17-05:00" level=error msg="Error while listing space quotas: Error requesting space quotas: cfclient: error (10000): CF-NotFound" source="spaces_collector.go:277"
  | time="2018-02-19T07:02:17-05:00" level=error msg="Error while listing stacks: Error requesting stacks: cfclient: error (10000): CF-NotFound" source="stacks_collector.go:142"
  | time="2018-02-19T07:02:17-05:00" level=error msg="Error while listing services: Error requesting services: cfclient: error (10000): CF-NotFound" source="services_collector.go:142"
  | time="2018-02-19T07:02:17-05:00" level=error msg="Error while listing security groups: Error requesting sec groups: cfclient: error (10000): CF-NotFound" source="security_groups_collector.go:142"
  | time="2018-02-19T07:02:17-05:00" level=error msg="Error while listing service planss: Error requesting service plans: cfclient: error (10000): CF-NotFound" source="service_plans_collector.go:142"
  | time="2018-02-19T07:02:18-05:00" level=error msg="Error while listing organization: Error requesting orgs: cfclient: error (10000): CF-NotFound" source="applications_collector.go:203"
  | time="2018-02-19T07:02:18-05:00" level=error msg="Error while listing organization quotas: Error requesting org quotas: cfclient: error (10000): CF-NotFound" source="organizations_collector.go:292"
  | time="2018-02-19T07:02:18-05:00" level=error msg="Error while listing routes: Error requesting routes: cfclient: error (10000): CF-NotFound" source="routes_collector.go:142"
  | time="2018-02-19T07:02:18-05:00" level=error msg="Error while listing service bindings: Error requesting service bindings: cfclient: error (10000): CF-NotFound" source="service_bindings_collector.go:142"

```
Could you please help us.

With Regards, 
Aparna Reji

**Comment by SamwiseGambgee** [comment]
Hey,
Make sure you remove the proxies before running
`export https_proxy=`
`export http_proxy=`

**Comment by aparnareji** [comment]
Hi , 

Removing the proxies didn't resolved the problem and the error is still there as same. May i know what else could be the problem.

With Regards,
Aparna Reji

**Comment by aparnareji** [comment]
Hi, 

Could you please provide me with an update.

With Regards,
Aparna Reji

**Comment by frodenas** [comment]
Try removing the trailing slash from the `CF_EXPORTER_CF_API_URL`. It should be `https://api.sys.adp.allianz`.

**Comment by aparnareji** [comment]
Hi @frodenas , 

The exporter is now working fine after removing the trailing slash from the CF_EXPORTER_CF_API_URL
Thank you 

With Regards,
Aparna Reji

**Comment by frodenas** [comment]
Thanks @aparnareji for confirming is working fine after removing the trailing slash. This should be fixed at [v0.6.2](https://github.com/bosh-prometheus/cf_exporter/releases/tag/v0.6.2).
